etrago/cluster/electrical.py

```diff
@@ -688,13 +694,15 @@ def postprocessing(etrago, busmap, medoid_idx=None):
         line_length_factor=settings["line_length_factor"],
     )
 
-    if method == "kmedoids-dijkstra":
-        for i in clustering.network.buses[clustering.network.buses.carrier == "AC"].index:
+    if method == "kmedoids-dijkstra" and len(medoid_idx) > 0:
```

Comment:
There is one case: If you use a csv-file instead of conducting the k-medoids Dijkstra Clustering yourself. 
Unfortunately, for this case, in the aggregation process there are no medoids, so in the end a the geographical mean per cluster defines the coordinations of the representative node. That's not perfectly nice, maybe we should also change that?